server/requirements.txt

```diff
@@ -1,4 +1,6 @@
 connexion[swagger-ui]==2.7.0
 pandas>=1.1.3
 python_dateutil==2.8.1
-swagger-ui-bundle==0.0.8
\ No newline at end of file
+swagger-ui-bundle==0.0.8
```

Comment:
The content of this file must be listed in ascending order.

Reply:
resolved